docs/macos-desktop-app.md

```
@@ -0,0 +1,377 @@
+| ---------------------- | ---------------------------------------------------- |
+| `expo-router`          | `@react-navigation/native` + macOS-native navigation |
+| `expo-secure-store`    | `react-native-keychain`                              |
+| `expo-file-system`     | `react-native-fs`                                    |
+| `expo-image-picker`    | `react-native-image-picker`                          |
+| `expo-document-picker` | `react-native-document-picker`                       |
+| `expo-haptics`         | Not needed on macOS                                  |
+| `expo-network`         | `@react-native-community/netinfo`                    |
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)